packages/venia-ui/lib/components/Image/image.js

```diff
@@ -45,6 +46,7 @@ const Image = props => {
         type,
         width,
         widths,
+        ratio,
```

Comment:
I would expect that ratio should also work with the `SimpleImage` component as well. I don't want to get into a huge rabbit hole re-factor, but the `useImage` talon seems like the best place to have this logic; if height isn't provided but ratio is, calculate height and drill that down.

Reply:
I haven't thought of that use case. Is it possible? Neverthless, it is good to have a check. Done. Nice suggestion @tjwiebell. Fixed in 5536d6156d5d4546aa56a7914e76a764e10464b3